packages/ui-sales/src/deals/containers/product/ProductForm.tsx

```
@@ -65,13 +67,13 @@ export default withProps<Props>(
     graphql<{}, ProductCategoriesQueryResponse, {}>(
       gql(queries.productCategories),
       {
-        name: 'productCategoriesQuery',
+        name: "productCategoriesQuery",
       }
     ),
     graphql<{ dealQuery: IDeal }, ProductCategoriesQueryResponse, {}>(
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)